libdnf/dnf-sack.c

```diff
@@ -1090,6 +1090,8 @@ dnf_sack_add_cmdline_package(DnfSack *sack, const char *fn)
     HyRepo hrepo = repo->appdata;
     hrepo->needs_internalizing = 1;
     priv->provides_ready = 0;    /* triggers internalizing later */
+    priv->considered_uptodate = FALSE;
```

Comment:
According that DNF was requested to apply excludes also for commandline repo, therefore I think it has a sense to keep it there.